pkg/manager/member/tikv_upgrader.go

```diff
@@ -491,7 +496,7 @@ func getStoreByOrdinal(name string, status v1alpha1.TiKVStatus, ordinal int32) *
 	return nil
 }
 
-func isTiKVReadyToUpgrade(tc *v1alpha1.TidbCluster) (bool, string) {
+func isTiKVReadyToUpgrade(pdControl pdapi.PDControlInterface, tc *v1alpha1.TidbCluster) (bool, string) {
```

Comment:
Similar to the `IsClusterStable`, maybe we can make some improvements and return the reason only? Another thing is, can we make this method a member function of `tikvUpgrader`?